README.md

```diff
@@ -33,6 +33,8 @@ Happy hacking!
 
 * Point browser to [localhost:5000](http://localhost:5000)
 
+* Note: the project overrides your Node version using .nvmrc which points to the latest Node LTS version.
```

Comment:
It only overrides if they are using nvm themselves. I'd be tempted to simply have a requirements section, something like:

```
Requirements

* Node v6.11.4 (LTS) 
*Recommended to use [NVM](https://github.com/creationix/nvm)
```

Reply:
Sounds good! I just changed this on the README.

Reply:
BWT, I did change this, but the commit history on the PR is not updating according to my `master` branch of the fork. If a couple of hours pass and this issue still exists, I'll make a new PR.